src/App.js

```diff
@@ -38,24 +42,29 @@ function App() {
             <ErrorBoundary>
               <Switch>
                 <Route
-                    path={Routes.SING_UP}
-                    component={SingUp}
-                    layout={MainLayout}
+                  path={Routes.PRIVATE_ROUTE}
```

Comment:
Add route for public and private router component, in which you will check JWT token

Reply:
added